docs/usage/faq.md

```diff
@@ -5,60 +5,64 @@ description: Frequently Asked Questions for Renovate Configuration
 
 # Frequently Asked Questions (FAQ)
 
-## What Is The Default Behavior?
+## What is the default behavior?
 
 Renovate will:
 
 - Look for configuration options in a configuration file (e.g. `renovate.json`) and in each `package.json` file
-- Find and process all package files (e.g. `package.json`, `package.js`, `Dockerfile`, etc) in each repository
+- Find and process all package files (e.g. `package.json`, `can-we-change-this-item?`, `Dockerfile`, etc) in each repository
```

Comment:
I think that having `package.js` right next to `package.json` looks like a spelling error (it was not immediately obvious that this is the format for the Meteor package manager.

Could we maybe use `composer.json` or something else instead? That way it doesn't look like a typo.

Reply:
Yes, that's from an old package manager that very few use today. Replace it with `composer.json`